frontend/tailwind.config.js

```diff
@@ -1,9 +1,5 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  corePlugins: {
-    // TODO: Readd after removing bulma base styles
-    preflight: false,
```

Comment:
Doesn't tailwind 4 come with something similar?

Reply:
It does! (though not as pretty)

Added now.